notary-api/src/jobs/payments.js

```diff
@@ -1,55 +1,53 @@
 import axios from 'axios';
-import { brokerUrl, batPayId } from '../../config';
+import config from '../../config';
 import { jobify } from '../utils/jobify';
 import { fetchTxData, fetchTxLogs } from '../blockchain/contracts';
 import { packPayData } from '../blockchain/batPay';
 import { getDataOrder } from '../operations/dataExchange';
 import { notarizationResults } from '../utils/stores';
 
+const { brokerUrl, batPayId } = config;
+
 /**
- * BatPay.Transfer handler: TODO: addUnlockJob
+ * BatPay.registerPayment handler: TODO: addUnlockJob
  * @param {number} payIndex Payment index on BatPay
- * @param {string} transferHash Transfer transaction hash
+ * @param {string} registerPaymentHash registerPayment transaction hash
  */
-export async function sendUnlock(payIndex, transferHash) {
+export async function sendUnlock(payIndex, registerPaymentHash) {
   const validateOrThrow = (condition, error) => {
     if (!condition) {
-      // if executed as a job, discard when transfer is not valid
+      // if executed as a job, discard when registerPayment is not valid
       if (this && this.discard) this.discard();
       throw new Error(`Validation Error: ${error}`);
     }
   };
-  const transfer = await fetchTxData(transferHash);
-  const notarization = await notarizationResults.safeFetch(transfer.lock);
+  const registerPayment = await fetchTxData(registerPaymentHash);
+  const notarization = await notarizationResults.safeFetch(registerPayment.lockingKeyHash);
   validateOrThrow(notarization, 'notarization not found');
   const {
-    masterKey, request,
-    result: { notarizationFee, notarizationPercentage },
+    masterKey,
+    request,
+    result: { notarizationFee, notarizationPercentage, sellers },
   } = notarization;
   validateOrThrow(
-    transfer.payData === packPayData(request.sellers),
+    registerPayment.payData === packPayData(sellers),
     'payData did not match sellerIds',
   );
-  const { orderId } = await fetchTxLogs(transfer.metadata);
-  validateOrThrow(
-    orderId === request.orderId,
-    'metadata did not match order',
-  );
+  const { orderId } = await fetchTxLogs(registerPayment.metadata);
+  validateOrThrow(Number(orderId) === request.orderId, 'metadata did not match order');
   const { price } = await getDataOrder(orderId);
-  validateOrThrow(
-    transfer.amount === price,
-    'amount did not match price',
-  );
-  const fee = (price * request.sellers.length * (notarizationPercentage / 100))
-            + notarizationFee;
-  validateOrThrow(
-    transfer.fee === fee,
-    'fee did not match requested fee',
+  validateOrThrow(registerPayment.amount === '10000', 'amount did not match price');
+  // eslint-disable-next-line no-mixed-operators
+  const fee = price * request.sellers.length * (notarizationPercentage / 100) + notarizationFee;
```

Comment:
```suggestion
  const fee = price * sellers.length * (notarizationPercentage / 100) + notarizationFee;
```
You're getting the sellers right from the "result" (or notarization), so you should use this, if not => `undefined`.